index.js

```diff
@@ -22,12 +22,13 @@ var safeBufferEquals = require('./lib/safe-buffer-equals')
 var replicate = null
 
 var defaultCrypto = {
+  keyPair: crypto.keyPair,
   sign (data, sk, cb) {
     return cb(null, crypto.sign(data, sk))
   },
 
-  verify (sig, data, pk, cb) {
-    return cb(null, crypto.verify(sig, data, pk))
+  verify (data, sig, pk, cb) {
+    return cb(null, crypto.verify(data, sig, pk))
```

Comment:
Was the swapping of args here intentional?

Reply:
Yes, to match the api of hypercore-crypto